pandas/tests/groupby/methods/test_cum.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
```

Comment:
in tests.frame and tests.series with have test_cumulative.py.  can we use that pattern?  and if we're really trying to follow the pattern, that file goes outside the methods/ directory